src/gameLogic.js

```diff
@@ -19,8 +19,8 @@ function whereCanMove(squares, piece, x, y) {
 	};
 	const addIfEmpty = (x, y, isEmpty) => addIfEmptyExt(x, y, true, isEmpty);
 	const addAllIfEmpty = (all) => {
-		for (let [x, y] of all)
-			addIfEmptyExt(x, y, isValidPosition(x, y));
+		for (const [a, b] of all)
```

Comment:
the variable names should be x and y, **not a and b**

Reply:
marked it as resolved because I added the 83bedeffae0a585d088c3465c1600902675ba915 commit